mitiq/benchmarks/mirror_circuits.py

```diff
@@ -149,7 +149,7 @@ def generate_mirror_circuit(
             ``cirq.Circuit``.
 
     Returns:
-        A randomized mirror circuit.
+        A randomized mirror circuit and correct bitstring.
```

Comment:
what does _correct_ mean here?

Reply:
Apologies, I accidentally committed this one to the wrong branch. It was intended as part of #2353, but I failed to notice I hadn't successfully switched branches. Mirror circuits are designed such that there is only one expected bitstring, and all others can be considered due to noise.